src/MLJGLMInterface.jl

```diff
@@ -187,7 +188,10 @@ function MMI.fit(model::LinearRegressor, verbosity::Int, X, y)
     features = glm_features(model, X)
     data = glm_data(model, Xmatrix, y, features)
     form = glm_formula(model, features)
-    fitresult = GLM.glm(form, data, Distributions.Normal(), GLM.IdentityLink(); offset=offset)
+    fitresult = GLM.glm(
+        form, data, Distributions.Normal(), GLM.IdentityLink();
+        offset=offset
+    )
```

Comment:
```suggestion
    fitresult = GLM.glm(form, data, Distributions.Normal(), GLM.IdentityLink(); offset)
```
It looks like the change was done via an automated formatter? Those are not perfect either. After changing `offset=offset` to `offset` the line length is within proper length.

Reply:
Yeah, It just occurred to me that this is perfect since we just set the minimum julia version used in ci to 1.6